docs/options.md

```diff
@@ -510,6 +533,12 @@ optional arguments:
                         don't run on your development machine. For "windows",
                         you need to run in Windows, and it will build and test
                         for all versions of Python. Default: auto.
+  --archs ARCHS         Comma-separated list of CPU architectures to build
+                        for. If unspecified, builds the architectures natively
```

Comment:
Unspecified is shown underneath by "default". It's more important to link it to the fact that that's what `auto` means.

Then again, that should be reasonably clear...

Reply:
Good point. I'm changing it to 

```
Comma-separated list of CPU architectures to build for.
When set to 'auto', builds the architectures natively supported
on this machine. Set this option to build an architecture
via emulation, for example, using binfmt_misc and QEMU. Default: auto. Choices: ...
```